### ISSUE TITLE
Breaking change in 7.0.1 - different positioning of items than in 7.0.0 (possibly outerMargin property ignored)

### ISSUE THREAD

**Issue by mc-suchecki** [tiberiuzuld/angular-gridster2#443]
It seems like in package version 7.0.1 property outerMargin set to false is ignored. It is a breaking change if I am not mistaken.

For the following gridster config:
```
 public readonly GRIDSTER_CONFIG: GridsterConfig = {
    gridType: GridType.VerticalFixed,
    outerMargin: false,
    margin: 30,
    draggable: {
      enabled: true,
      ignoreContentClass: 'not-draggable',
      dragHandleClass: 'panel-heading'
    },
    fixedRowHeight: 250,
    pushItems: true,
    swap: true,
    minCols: 2,
    maxCols: 2,
    compactType: 'compactUp',
    itemChangeCallback: this.onGridItemMove.bind(this)
  };
```

Styles for gridster-item on 7.0.0:
```
element.style {
    display: block;
    height: 530px;
    transform: translate3d(0px, 0px, 0px);
    width: 698px;
}
```

Styles for gridster-item on 7.0.1:
```
element.style {
    display: block;
    height: 530px;
    left: 30px;
    top: 30px;
    width: 787.5px;
}
```

Notice the ```left``` and ```top``` properties added. This is messing up my layout and I believe should be fixed or reported as a breaking change.

Tested in Firefox, Chrome, Edge & IE.

**Comment by tiberiuzuld** [comment]
Fixed in v7.1.0
Thanks

**Comment by mc-suchecki** [comment]
Thank you!
